test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -220,15 +178,15 @@ public void GeneratePreContent_ReturnsNothingIfSelfClosingWhenTagNameIsNotNullOr
             // Arrange
             var tagHelperOutput = new TagHelperOutput("p")
             {
-                SelfClosing = true,
-                PreContent = "Hello World"
+                SelfClosing = true
```

Comment:
teensy nit: in future don't remove the trailing initializer commas.  you're more likely to get nits to _add_ them.